azure-pipelines.perf-test.yml

```diff
@@ -12,26 +12,25 @@ steps:
   # No need for checkout since we're using build artifacts.
   - checkout: none
 
-  # TODO: why is this different from main pipeline? is this really getting the correct artifacts?
-  # I think this could be grabbing the wrong (most recent PR) artifacts rather than the one it's associated with
-  # This run is for PR 20200219.5 build #53232 but is grabbing artifacts from 2020219.6 build #53235
-  # https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=53238&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=559759ef-66f4-5335-a959-132505aae30a
-  # Last PR Build: https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=53232&view=logs&jobId=92058b04-f16a-5035-546c-cae3ad5e2f5f&j=ca395085-040a-526b-2ce8-bdc85f692774&t=25a453a0-e716-5e6e-0bee-a24e8e541880
   - template: azure-pipelines.artifacts.yml
     parameters:
       artifact: $(artifact)
       buildId: $(buildId)
+      # $BUILD_REASON for this pipeline will always be Manual. buildReason must be passed in
+      # from triggered pipeline so that this pipeline knows the originating build reason.
+      buildReason: $(buildReason)
```

Comment:
@kenotron fyi, just wanted you to sanity check this. The Fluent pipeline takes actions on PR vs. Master builds and I had to pass in the build reason from the originating pipeline (since `$BUILD_REASON` is always `Manual` here.)